modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/test/HeadlessBuilderTest.java

```diff
@@ -113,6 +117,35 @@ InfoItemFormProvider.class, new TestEntryInfoItemFormProvider(),
 					false)),
 			ObjectDefinitionConstants.SCOPE_COMPANY,
 			TestPropsValues.getUserId());
+
+		InputStream inputStream = getClass().getResourceAsStream(
+			"/rest-openapi.yaml");
+
+		Scanner scanner = new Scanner(inputStream);
+
+		while (scanner.hasNextLine()) {
+			_finalOpenAPI = StringBundler.concat(
```

Comment:
Is it possible to use `com.liferay.portal.kernel.util.StringUtil.read(InputStream inputStream)`?

Reply:
True! I didn't know the existence of that method. I've already changed it. Thanks!